docs/content/singleton-actors.md

```diff
@@ -6,14 +6,14 @@ Singleton Actors allow a single source of truth for both operations and data acr
 
 There are two kinds of Singleton Actors:
 
-* **Public Singleton Actors** - Singleton Actors which are replicated to clients. `GameState` is a Public Singleton Actor.
-* **Private Singletons** - Singleton Actors which are not accessible to clients. `GameMode` is a Private Singleton Actor.
+* **Public Singleton Actors** - Singleton Actors which are replicated to client-workers. `GameState` is a Public Singleton Actor.
+* **Private Singletons** - Singleton Actors which are not accessible to client-workers. `GameMode` is a Private Singleton Actor.
```

Comment:
...accessible 'by' client-workers?

Reply:
Hmm, I think either works. But I just asked Google (as you do) and:

- "accessibly by" has 17,500,000 results
- "accessible to" has 52,500,000 results

Reply:
TIL.

Reply:
Also could we do a drive-by on these sentences? We aren't stating that by default all singletons are accessible by all server-workers.
Perhaps:
public: `Singleton Actors which are replicated to server-workers and client-workers`
private: `Singleton Actors which are replicated to server-workers, but not accessible to client-workers`
?

Reply:
Do you mean we _should_ be saying that by default all singletons are accessible to all server-workers? If so, this looks good.

Reply:
Yes. Would you mind applying these changes to this PR?

Reply:
Late to the party - accessible *to* for sure.
Yes, be explicit in the docs even if it seems obvious so your additions look good @m-samiec.